src/pybind/mgr/volumes/fs/stats_util.py

```diff
@@ -89,7 +89,7 @@ def __init__(self, volclient, volspec):
 
         # Creating an RTimer instance in advance so that we can check if clone
```

Comment:
Comment needs to remove reference of clone as it's generic now, please do the same in below comments as well.

Reply:
Fixed.